### ISSUE TITLE
Have placeable locations for cards light up without needing to hover over them.

### ISSUE THREAD

**Issue by Luk-Er4** [edogdu/RAM_Cyber_Defenders#48]
Some players might have trouble figuring out where to put the cards. Lighting up the placeable locations can help the players to keep playing the game.

**Comment by locada1** [comment]
Glowing models with simple animations have been added to the board to highlight where a blue card, green card, or red card can be placed, or when a card will need to be placed in the discard pile. This works for both players' cards, and these card guides can currently be turned off or on via a UI button to the side. 

Below are gifs of the blue, green, and red cards. (The panel to the right of the AI displays the card it drew on it's last turn)

![Blue Card - Made with Clipchamp](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/ecee4d6b-be45-4010-ad41-5bee8dacb0a0)

![Green Card - Made with Clipchamp](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/44d74a86-9623-4c98-a0d2-8e4700e314b6)


![Red Card - Made with Clipchamp](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/f7242808-058a-47a6-83f7-20e7d3fc3e08)

**Comment by locada1** [comment]
I have made it so the card animations on the board play once you or the opponent have drawn their card.
This currently works by playing the animations when the next card is not on the draw pile.

![Card Animations - Made with Clipchamp](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/f5ca2f71-01b5-4e03-a23a-313d9b519999)

**Comment by locada1** [comment]
I have added an in-game option to have card guides and their animations to either always play or only play when the next card is drawn by either the player or AI.

I have added a menu holding all the options to change tips and card guides settings to the right of the player's position.

![CS4091 Beta 1 4 - SecondRerrangement - Windows, Mac, Linux - Unity 2022 3 22f1_ _DX11_ 7_7_2024 5_18_52 PM](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/a7201058-ee40-456d-9c1f-d8f54aacd0da)

![CS4091 Beta 1 4 - SecondRerrangement - Windows, Mac, Linux - Unity 2022 3 22f1_ _DX11_ 7_7_2024 5_19_00 PM](https://github.com/edogdu/RAM_Cyber_Defenders/assets/145424979/72e5db63-30a3-4fdc-ae8e-7aaca1e021e2)

[ Improvements and better clarifications for this menu will be made later ]


This is what the buttons do currently:

Disable Tips - Turns off any active tips windows and will prevent any new tips windows from showing for that match.

Disable Guide - The card guides and their animations will be disabled; this button can be pressed again to re-enable card guides. (Card guides are on by default)

Always Show Guide - Cards guides will be set to always show on the board; this button can be pressed again to only show card guides when a card is drawn (this is the default setting).
